mkdocs.yml

```diff
@@ -55,6 +55,7 @@ nav:
         - Statuses: 'models/extras/status.md'
         - Tags: 'models/extras/tag.md'
         - Webhooks: 'models/extras/webhook.md'
+        - Relationships: 'models/extras/relationships.md'
```

Comment:
Additional Features are listed in alphabetical order, so please move this to between `Prometheus Metrics` and `Statuses` in the listing.